services/frontend/src/resources/css/style.scss

```
@@ -11,3 +11,131 @@ $custom-colors: (
 );
 
 @import '~@creativebulma/bulma-tooltip/dist/bulma-tooltip.css';
+
+#historyModal .history-modal-card {
+  width: min(1200px, 95vw);
+  max-width: none;
+  // Anchor the modal at a fixed height. Without this, the modal sizes to its
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)